src/pykiso/lib/robot_framework/communication_auxiliary.py

```diff
@@ -60,9 +61,31 @@ def send_message(
         logger.info(f"send message {raw_msg} using {aux_alias}")
         return state
 
+    @keyword(name="Start Recording")
```

Comment:
How do I know what I start recording? (maybe I lack of robot framework understanding)